modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js

```diff
@@ -70,10 +53,8 @@ describe('CardShortcut', () => {
 	it('anchors are created for each field', () => {
```

Comment:
@brunofariasdeo, please update the test's description according to your changes. Thanks.

Reply:
@natocesarrego, I have updated it accordingly to my changes, I have missed the fact that 'anchor' stands for 'a' tag.

Reply:
No problem. Thanks. 🙂